### ISSUE TITLE
Sloeber menu not visible on first launch

### ISSUE THREAD

**Issue by Alex-Kent** [Sloeber/arduino-eclipse-plugin#1580]
When launching Sloeber for the first time (when built from source) the **Sloeber** menu does not appear in the menu bar:

![No menu](https://github.com/Sloeber/arduino-eclipse-plugin/assets/44915283/51c82d4b-d545-4d56-a8fb-c62bacb7640d)

Once the following menu item is selected the **Sloeber** menu becomes visible:

    Window → Perspective → Open Perspective → C/C++

![Menu](https://github.com/Sloeber/arduino-eclipse-plugin/assets/44915283/18c3dc46-3dc9-469a-92f8-03130e898e5a)

In other words the C/C++ perspective should be active by default.

---

Further note: Creating a new Arduino Sketch (using `File → New → Other… → Arduino Sketch` or `File → New → Other… → Sloeber → Arduino Sketch`) also does not activate the C/C++ perspective.   (Building the sketch works fine though.)

**Comment by Alex-Kent** [comment]
I tried adding `org.eclipse.ui/defaultPerspectiveId=org.eclipse.cdt.ui.CPerspective` to `io.sloeber.product/plugin_customization.ini` followed by a build using `mvn clean verify -Plinux64,latest,NOSDK -DskipTests=true` but it didn't fix things.

The test was run under a test user (`test`) with `SLOEBER_HOME` set to `/home/test` and no `~/.eclipse` or `~/Documents/sloeber-workspace` directories.

**Comment by jantje** [comment]
The automagic selecting of the Sloeber perspective is something I never got to work.
There are several places that it says the Sloeber perspective should be selected by default; however it is not.
To me this is just one of these things that are supposedly simple but are not.

**Comment by Alex-Kent** [comment]
Not a big deal, just something I noticed.  If I'm reading the docs correctly the fix I tried should have worked...

**Comment by jantje** [comment]
I wish it worked. It is most problematic for people not being used to the eclipse perspective as they do not see any Sloeber stuff

**Comment by jantje** [comment]
FYI: the Sloeber menu is only visible in C/C++ perspective or Sloeber perspective. That is "as designed".
